back/config/initializers/register_admin_publication_filters.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+AdminPublicationsFilteringService.add_filter('by_folder') do |scope, options|
```

Comment:
I think we can just add this to the admin publication filtering service. Perhaps we could even consider to remove the `add_filter` method?